tests/models/t5/test_modeling_t5.py

```diff
@@ -1607,11 +1641,71 @@ def test_contrastive_search_t5(self):
         self.assertListEqual(
             generated_text,
             [
-                "Liana Barrientos has been married 10 times, nine of them in the Bronx. Her husbands filed for "
-                "permanent residence after the marriages, prosecutors say."
+                "Liana Barrientos has been married 10 times, nine of them in the Bronx . Her husbands filed for "
+                "permanent residence after the marriages, prosecutors say ."
```

Comment:
Is this a new failure? 

If yes: can you confirm a) the generated tokens are different b) compare the logprobs of the different tokens before/after the change, to see whether it can be explained by tiny fluctuations?

If no: can you add a comment with the PR that causes the difference? 🙏

Reply:
Added a comment, the regression was caused by changes in the tokenization. So nothing breaks in terms of modeling or generation. Not sure if we have to the fix in tokenization was intended to break (i.e. fix this) or we need another PR to fix tokenization back, so I'll leave a comment for future us